### DISCUSSION TITLE
No Bluetooth Light or Connection

### DISCUSSION THREAD

**Discussion by xLionTamer** [iRobotEducation/create3_docs#256] [Questions] [answered]
I received the Create3, updated the firmware (G.3.1), and joined it to my wireless network. I am attempting to connect to it through the Python Web Playground in the Chrome browser. It does not show as a device when I click on Connect. 
I did not see a blue bluetooth indicator light, so I removed the faceplate. The adapter board showed the toggle flipped to bluetooth. I flipped it to USB and back to bluetooth. I still do not see a blue indicator light and cannot connect over bluetooth.
I restarted the device, but that did not help. Please let me know what else I can do to diagnose the connection.

**Comment by shamlian** [comment, marked as answer]
First, check to see if the adapter board is fully seated within the robot. Remove the payload bay, then pull out the adapter board, check the connector for debris, and then push it back in.

If the blue indicator light doesn't illuminate when the USB/BLE switch is set to Bluetooth while the robot is powered on, I'd suggest checking that VBAT is available on the JST XH header, using a multimeter. If not, I suspect something is wrong with the robot; if so, I suspect something is wrong with the adapter board. It would be helpful to see logs from the robot after toggling the switch from USB to BLE, with the robot booted. Either way, you should contact education@irobot.com, referencing this post, because likely something is wrong with the hardware, and someone from support will be able to help.

**Reply by shamlian** [reply]
One silly addition -- just to be sure -- is the adapter board fully seated within the robot? I've heard that it can get nudged out during shipping. The tab on the bottom of the adapter board should be fully bottomed out against the back of the robot.

**Comment by xLionTamer** [comment]
It did feel loose. I will look when I have it in front of me. Do you have any tips for re-seating the board?

**Reply by shamlian** [reply]
With the cargo bay removed, you should be able to grasp the back of the board and pull it out; it's on a blind mate edgecard connector. Just pull it out, inspect the business end for any debris (think NES cartridge) and then shove it back in until it won't go anymore. If you also take the faceplate off, you'll be able to see the PCB it's mating to, as well.

**Comment by xLionTamer** [comment]
Success! Light is on and bluetooth communication is working! I appreciate the support.
